lib/datasource/remote/api/network_repository.dart

```diff
@@ -77,3 +78,11 @@ class NetworkError extends Error {
 
   NetworkError(this.statusCode);
 }
+
+class MyHttpOverrides extends HttpOverrides {
+  @override
+  HttpClient createHttpClient(SecurityContext? context) {
+    return super.createHttpClient(context)
```

Comment:
If we have to go with this solution - if it's the only one that works - then we should limit the "true" response to only hosts that we know. I think we are accessing nodes, right, so we can see if the access is to one of our nodes, and then allow it. 

Not for all requests which could have unforeseen consequences and lowers the overall security of our app.